src/main/java/org/gridsuite/sensitivityanalysis/server/SensitivityAnalysisController.java

```diff
@@ -144,4 +145,18 @@ public ResponseEntity<Double> getDefaultResultsThreshold() {
         return ResponseEntity.ok().body(service.getDefaultResultThresholdValue());
     }
 
+    @GetMapping(value = "/providers", produces = APPLICATION_JSON_VALUE)
+    @Operation(summary = "Get all sensitivity analysis providers")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "Sensitivity analysis providers have not been found")})
+    public ResponseEntity<List<String>> getProviders() {
+        return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON)
+                .body(service.getProviders());
+    }
+
+    @GetMapping(value = "/default-provider", produces = TEXT_PLAIN_VALUE)
+    @Operation(summary = "Get sensitivity analysis default provider")
+    @ApiResponses(@ApiResponse(responseCode = "200", description = "The sensitivity analysis default provider has not been found"))
```

Comment:
The sensitivity analysis default provider has been found

Reply:
Done